tutorials/cp-kyma-redis-function/cp-kyma-redis-function.md

```diff
@@ -39,6 +38,26 @@ This sample provides a Redis deployment and two Serverless Functions that intera
     git clone https://github.com/SAP-samples/kyma-runtime-extension-samples
     ```
 
+### Add the Serverless, Eventing and NATS modules in SAP BTP, Kyma runtime
+
+1. Open Kyma dashboard using the **Console URL** link in SAP BTP cockpit.
```

Comment:
The "Console" will be replaced with something else soon, right?  but it is not applicable for now yet I guess.

Reply:
Yes, I keep that in mind. As soon as it is changed, I'll fix that.